lib/ansible/modules/files/find.py

```diff
@@ -33,7 +33,7 @@
         default: '*'
         description:
             - One or more (shell or regex) patterns, which type is controlled by C(use_regex) option.
-            - The patterns restrict the list of files to be returned to those whose basenames match at
+            - The patterns restrict the list of files to be returned to those whose complete filenames match at
```

Comment:
at the very least this should note that this changed across versions and the behaviour in each case

Reply:
I updated the documentation to  add when this feature was added.

I'm not sure how can I add in the doc the behaviour in each case. 
Do you have an example of another module documenting this kind of things ?

Reply:
quite a few, ini_file, ec2_ami_copy, ec2_eni, ovirt_hosts .. just to list a few